docs/reading_files.rst

```diff
@@ -146,6 +146,11 @@ below, e.g.::
    
       .. versionadded:: 1.7
 
+   .. automethod:: as_single_value
```

Comment:
I've roughly grouped the reading methods before the selection methods, if you want to move this up - maybe just after `dask_array`, to keep the various array-like methods together?

Reply:
I don't mind, but I put it down there exactly because we concluded it's not just a reading method 🙃

Reply:
Not *just* a reading method, but it's more like `.ndarray()` than `.select_trains()` - it gives you back data rather than another KeyData object.

Dask arrays kind of mess up this categorisation, though, so I don't feel strongly about it.